docs/sources/mimir/references/architecture/components/alertmanager.md

```diff
@@ -15,6 +15,8 @@ The Mimir Alertmanager adds multi-tenancy support and horizontal scalability to
 The Mimir Alertmanager is an optional component that accepts alert notifications from the [Mimir ruler]({{< relref "./ruler" >}}).
 The Alertmanager deduplicates and groups alert notifications, and routes them to a notification channel, such as email, PagerDuty, or OpsGenie.
 
+> **Note:** To run Mimir Alertmanager as a part of [monolithic deployment]({{< relref "/../deployment-modes/#monolithic-mode" >}})run mimir with the command `./mimir -target=all,alertmanager`.
```

Comment:
I appreciate you reading the documentation on linking but choosing to remain consistent within the project, I think that was the right decision! Thank you :)

I have some suggestions for this specific note and I'll ask a project maintainer to check that it is encouraged to run a monolithic deployment with Alertmanager in this way.

Suggestions for the note:
- Use the [`admonition` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition) which has consistent styling on the website.
  The project uses blockquotes and the shortcode inconsistently at the moment but I've opened https://github.com/grafana/mimir/pull/7420 to resolve that inconsistency.
- Fix the link which had a typo.
- Remove the explicit invocation of the binary since it might be somewhere else in the file system.

```suggestion
{{< admonition type="note" >}}
To run Mimir Alertmanager as a part of [monolithic deployment]({{< relref "../deployment-modes#monolithic-mode" >}}), run Mimir with the option `-target=all,alertmanager`.
{{< /admonition >}}
```